crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -305,7 +307,14 @@ def disable_non_core_proposals
     end
 
     # and finally delete the roles itself
-    active_non_core_roles.each(&:destroy)
+    roles_to_disable.each do |role|
+      if role.name == "crowbar-config-default"
+        role.elements["crowbar-upgrade"] = []
+        role.save
+      else if role.name != "crowbar-upgrade"
+        role.destroy
+      end
+    end
```

Comment:
Yes... the point is, I want that previous code where `node.delete_from_run_list` is executed to be run for all roles here, so also for crowbar-config-default and crowbar-upgrade